tests/menu/test_config.py

```diff
@@ -21,26 +22,10 @@ def test_config_restore():
     os.remove(BUILDTEST_CONFIG_BACKUP_FILE)
     func_config_restore()
 
+def test_valid_config_schemas():
 
-def test_config_local():
-    example_schema = os.path.join(
-        pytest_root, "config_schema_examples", "local-example.yml"
-    )
+    valid_schema_dir = os.path.join(pytest_root,"examples","config_schemas","valid")
```

Comment:
Does this validate with black?

Reply:
i ran black against all files, seems like line 27 ``valid_schema_dir`` didn't change so i guess it is valid with black.